include/sot/core/robot-utils.hh

```diff
@@ -305,6 +305,7 @@ public:
           << " for get_parameter(" << parameter_name << ")\n"
           << e.what() << std::endl;
       sendMsg(oss.str(), MSG_TYPE_ERROR);
+      return 0;
```

Comment:
@olivier-stasse Usually, I see an API like
```cpp
/// This throws is name is not found
template <typename Type> Type getParameter(std::string name);

/// This returns defaultValue if throws is name is not found
template <typename Type> Type getParameter(std::string name, Type defaultValue);
```

What is returned here is valueless and this is fairly hazardous. The `return 0;` do solve the compilation warning but not the underlying design flaw.

Reply:
Thanks for the comment
Is there another way to silence those warnings ? @jmirabel

Reply:
The other way requires to modify the API.

Reply:
I agree. The best strategy here would be to rethrow the exception.